test/basic-querying.test.js

```diff
@@ -220,6 +236,20 @@ describe('basic-querying', function() {
       });
     });
 
+    it('should query sorted desc by order integer field even though there' +
+       'is an async model loaded hook', function(done) {
+      User.find({ order: 'order DESC' }, function(err, users) {
+        if (err) return done(err);
+
+        should.exists(users);
+        should.not.exists(err);
+        users.pop().order.should.equal(1);
+        users.pop().order.should.equal(2);
+        users.shift().order.should.equal(6);
```

Comment:
When one of these three assertions fail, it's difficult to tell what exactly happened, as the error message shows only one item from the array.

I think this may be a better solution:

```js
var order = users.map(function(u) { return u.order; });
order.should.eql([1,2,3,4,5,6]);
```